services/spar/src/Spar/Sem/IdPConfigStore/Spec.hs

```diff
@@ -55,8 +55,8 @@ propsForInterpreter interpreter extract labeler lower = do
     prop "getConfig/storeConfig" $ prop_getStore (Just $ show . (() <$) . extract) lower
     prop "getConfig/getConfig" $ prop_getGet (Just $ show . ((() <$) *** (() <$)) . extract) lower
     prop "setReplacedBy/clearReplacedBy" $ prop_setClear labeler lower
-    prop "setReplacedBy/getReplacedBy" $ prop_setGet (Just $ show . (fmap (() <$)) . extract) lower
-    prop "setReplacedBy/setReplacedBy" $ prop_setSet (Just $ show . (fmap (() <$)) . extract) lower
+    prop "setReplacedBy/getReplacedBy" $ prop_setGet (Just $ show . fmap (() <$) . extract) lower
+    prop "setReplacedBy/setReplacedBy" $ prop_setSet (Just $ show . fmap (() <$) . extract) lower
```

Comment:
Isn't `(() <$)` the same as `void`?. I'm surprised hlint doesn't report this.

Reply:
Good point. Adding as a custom rule.